test/test.js

```diff
@@ -22,20 +22,39 @@ test('empty', () => {
 
 for (const id of Object.keys(expected.triangles)) {
 
-    test(id, () => {
-        const data = flatten(JSON.parse(fs.readFileSync(new URL(`fixtures/${id}.json`, import.meta.url)))),
-            indices = earcut(data.vertices, data.holes, data.dimensions),
-            err = deviation(data.vertices, data.holes, data.dimensions, indices),
-            expectedTriangles = expected.triangles[id],
-            expectedDeviation = expected.errors[id] || 0;
-
-        const numTriangles = indices.length / 3;
-        assert.ok(numTriangles === expectedTriangles, `${numTriangles} triangles when expected ${expectedTriangles}`);
-
-        if (expectedTriangles > 0) {
-            assert.ok(err <= expectedDeviation, `deviation ${err} <= ${expectedDeviation}`);
-        }
-    });
+    for (const rotation of [0, 90, 180, 270]) {
```

Comment:
I added this while debugging to test some adjacent cases, I can roll it it back if you don't want it.